### DISCUSSION TITLE
The --ignore CLI flag seems seems broken or am I misusing it?

### DISCUSSION THREAD

**Discussion by DanielFeichtinger** [swc-project/swc#7252] [Q&A]
I'm not sure if I'm using it correctly, but the --ignore flag for the CLI seems to add files to the compilation input rather than exclude them.

Using versions:
```
@swc/cli: 0.1.62
@swc/core: 1.3.49
```

Given a simple project with 4 .js files in ./src:

```
% npx swc ./src -d ./out           
Successfully compiled: 4 files with swc (13.1ms)
% npx swc ./src -d ./out --ignore src/foo.js 
Successfully compiled: 4 files with swc (12.64ms)
% npx swc ./src -d ./out --ignore **/*.js          
Successfully compiled: 1599 files with swc (983.52ms)
```

In the last case it's grabbing the contents of `node_modules` and compiling that too so looks to me like the pattern from --ignore is being used to add to the list of input files.

I can't see any issue or discussion on this but it's definitely unexpected behaviour — just wanted to make sure that I was using it correctly before filing anything!

**Comment by shawnmcknight** [comment]
I'm having the same exact issue and, like you, question whether I'm doing something wrong.  Your syntax is matching mine and _seems_ correct based on the docs.
